css/css-values/acos-asin-atan-atan2-invalid.html

```diff
@@ -67,10 +67,6 @@
 test_invalid_angle('atan2(1deg 2deg)');
 test_invalid_angle('atan2(1deg, , 2deg)');
 test_invalid_angle('atan2(90px)');
-
-// atan2() has to resolve to the same dimension.
-test_invalid_angle('atan2(90px, 90vw)');
-test_invalid_angle('atan2(90px, 90em)');
```

Comment:
If they should be valid, can we actually test their output?

Reply:
I filed on https://bugzilla.mozilla.org/show_bug.cgi?id=1802744